tests/integration/src/test/scala/mesosphere/marathon/integration/AppDeployIntegrationTest.scala

```diff
@@ -82,8 +82,67 @@ class AppDeployIntegrationTest extends AkkaIntegrationTest with EmbeddedMarathon
       queueAfterScaling.map(_.delay.overdue) should contain(false)
     }
 
+    "backoff delay can be get/deleted on a crash looping app" in {
+      Given("a crash looping app instance")
+      val id = testBasePath / "crash-looping-app-with-backoff-delay"
+      val uptime = 10.seconds
+      val app: App = createAFailingApp(id, Some(s"sleep ${uptime.toSeconds} && false"), uptime, 1.hour)
+
+      When("we try to fetch the current delay on the app")
+
+      val delayShownForQueuedInstance = () => {
+        val response = marathon.launchQueueDelayShow(id)
+        response should be(OK)
+        response.value.queueDelay shouldBe defined
+        response.value.currentDelaySeconds should be >= app.backoffSeconds.toLong
+        response.value.maxLaunchDelaySeconds shouldEqual app.maxLaunchDelaySeconds
+        true
+      }
+
+      val delayShownForHealthyInstance = () => {
+        val response = marathon.launchQueueDelayShow(id)
+        response should be(OK)
+        response.value.queueDelay shouldBe empty
+        response.value.currentDelaySeconds should be >= app.backoffSeconds.toLong
+        response.value.maxLaunchDelaySeconds shouldEqual app.maxLaunchDelaySeconds
+        true
+      }
+
+      val delayResetSuccess = () => {
+        val response = marathon.launchQueueDelayReset(id)
+        response should be(NoContent)
+        true
+      }
+
+      val patienceConfig: WaitTestSupport.PatienceConfig = WaitTestSupport.PatienceConfig(timeout = 5.minutes, interval = 2.seconds)
```

Comment:
necessary to do 2.seconds interval here? Maybe we can just use the default patienceConfig? (somewhat spamming API is okay by me if the test passes; higher frequency of polling will lead to faster completion)

Reply:
main reason for not using the default patienceConfig was to prevent usage of `Span` (if/when it auto scales) as we rely on the value of `X` in `sleep X` where we expect the app to be `RUNNING` and thus I wanted a constant `interval`.